x11-wm/xmonad-contrib/Manifest

```diff
@@ -1 +1,2 @@
 DIST xmonad-contrib-0.17.0.tar.gz 615173 BLAKE2B d2e2724a3103502cafeddfdafe5b19cd3f2eb66fe26d49afe4b64c0c78bb3b18432e44fc8233aac951b20969013fa580f0af36d4da627f3c03145b647bff5a44 SHA512 357ef80565690a1e652f79c877964181134c7f02a5d325dbf3890246588ce8a545e91126041bcc2af3d9ebf5b3be28f4866ee9ede76ddbf0fc0387871319a900
+DIST xmonad-contrib-0.17.1.tar.gz 646843 BLAKE2B 0949a616f6c5aaac5d5ccb8eacaf743898b8da6d0c99605ce2f7a85f8ce7747f5442843b524470d51300539126f25066b50906a92952129a9e264f18f7aa9873 SHA512 07f989623f4d9affae564ba36a180721910c1a0acf3d23e0e548fc48bb4dcd9864987c521c28251f6443cdd5134ded8e8bb18c6f78d06cb00c386ff6ad59c97e
```

Comment:
Like in https://github.com/gentoo-haskell/gentoo-haskell/pull/1322,

Please also remove `xmonad-contrib-0.17.0.ebuild`, update the manifest (`pkgdev manifest`), and amend/fixup/squash the changes to cf8f61f2a89dbb2b5f347e37e596b69903fea9f7.

This will

a. Remove the old version
b. Trigger a diff view in github between the two versions

Reply:
Got it, removed `xmonad-contrib-0.17.0.ebuild`.